test/browser/spec/ol/renderer/layer.test.js

```diff
@@ -7,15 +7,27 @@ import View from '../../../../../src/ol/View.js';
 import XYZ from '../../../../../src/ol/source/XYZ.js';
 import {fromKey} from '../../../../../src/ol/tilecoord.js';
 
-describe('ol.renderer.Layer', function () {
-  let renderer;
+describe('ol/renderer/Layer', function () {
+  let layer, renderer;
   const eventType = 'change';
 
   beforeEach(function () {
-    const layer = new Layer({});
+    layer = new Layer({});
     renderer = new LayerRenderer(layer);
   });
 
+  describe('#renderIfReadyAndVisible', function () {
+    it('updates revision when data is ready and layer is visible', function () {
+      layer.setVisible(true);
+      layer.getSourceState = function () {
+        return 'ready';
+      };
```

Comment:
To test for this regression it should return something other than `'ready'` and check whether the revision did not change. But it makes sense to test both cases.